assets/admin/editor-wizard/steps/lesson-details-step.js

```diff
@@ -1,21 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
 /**
  * Initial step for course creation wizard.
  *
  * @param {Object}   props
  * @param {Object}   props.data
  * @param {Function} props.setData
  */
-const LessonDetailsStep = ( { data, setData } ) => {
-	const onTitleChange = ( event ) => {
-		setData( { ...data, title: event.target.value } );
-	};
+const LessonDetailsStep = ( { data: wizardData, setData: setWizardData } ) => {
+	// Update modal title.
+	useEffect( () => {
+		setWizardData( { ...wizardData, modalTitle: 'Lesson Details Step' } );
+	}, [] );
```

Comment:
I feel like this is sounding like some sort of boilerplate, honestly, even if it's too small.

What do you think about adding the title as part of the component "static" properties, in the same way you did with the "Actions" static property? This way, we could change this whole `useEffect` by a simple additional static property:

```LessonDetailsStep.Title = 'Lesson Details Step'```

And then you can update the modalTitle appropriately when the steps change :)

Reply:
I really like the simplicity on this approach.

Change 0257a7b25.